accounts/serializers.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework import serializers
+from accounts.models import User
+from datetime import timedelta
+from django.utils import timezone
+class UserRegistrationSerializer(serializers.ModelSerializer):
+    password2 = serializers.CharField(style={'input_type':'password'}, write_only=True)
+    class Meta:
+        # For confirm password field in Registration Request
+        model = User
+        fields = ['email','name','password','password2']
+        extra_kwargs ={
+            'password': {'write_only':True}
+        }
+    # password and confirm password2 validation
+    def validate(self, attrs):
+        password = attrs.get('password')
```

Comment:
give approprate variable name i.e. password, confirm_password

Reply:
password2 renamed to confirm_password